script.js

```diff
@@ -130,58 +136,68 @@ $(document).ready(function() {
     $.ajax({
       url: urlquery,
       method: "GET"
-    }).then(function(response) {
-      if (response.status === 400) {
-        return $(".breedInfo").text(
-          "No hounds by the name of '" +
-            breed +
-            "' exist in this land! Double check the letters on your scroll!"
-        );
-      }
+    })
```

Comment:
all of the changes below were just spacing.. i used prettier to format so i didnt change any of jordans code